.words

```diff
@@ -0,0 +1,9 @@
+8
```

Comment:
surprised to see that a number isn't accepted by default by gospel.

Reply:
Duh, this always trips me up - it's just a length hint.

Does hunspell support comments? If so, it would be nicer to instead have `8 # remaining line number hint`

Reply:
So Phil did put comments in one that he generated, though they are not supported and would probably not work here. There is obviously the possibility of just breaking support for the format since we don't actually use it any more. Then it can be any format we want, though that complicates documentation because I can't just say, it's a .dic file.